docs/deployment.rst

```diff
@@ -96,23 +96,24 @@ Next we need to configure *aiohttp upstream group*:
 
    http {
      upstream aiohttp {
-       fail_timeout=0 means we always retry an upstream even if it failed
+       # fail_timeout=0 means we always retry an upstream even if it failed
```

Comment:
may be there should be two sections one for tcp server and for unix socket server